api/api_test.go

```diff
@@ -3,10 +3,9 @@ package api
 import (
 	"context"
 	"github.com/gorilla/mux"
+	. "github.com/smartystreets/goconvey/convey"
```

Comment:
Small suggestion but it's good to import the standard go libraries before external dependencies. 
The imports in `api/handlers.go` is a good example.